eng/templates/stages/release.yml

```diff
@@ -148,6 +152,33 @@ stages:
             draft='--draft'
           fi
 
+          git_user=dotnet-bot
+          git_email=dotnet-bot@microsoft.com
+
+          git config --global user.name "$git_user"
+          git config --global user.email "$git_email"
+
+          vmr_url=$(echo $vmrPublicUrl | sed "s,https://.*@,https://$(git_user):${GH_TOKEN}@,g")
+
+          git fetch $vmr_url tag $releaseTag --no-tags
```

Comment:
Curious, what does the `--no-tags` do?

Reply:
It stops `git` form pulling all tags. If I understood correctly it creates a local ref for the tag in question but pulls all associated tags (and lists them once the command is finished).

With this option set it will only pull the mentioned tag and will create a local ref for it

Reply:
Thanks